service/src/main/kotlin/fi/espoo/evaka/decision/DecisionController.kt

```diff
@@ -86,7 +86,7 @@ class DecisionController(
         @PathVariable("id") decisionId: UUID
     ): ResponseEntity<ByteArray> {
         Audit.DecisionDownloadPdf.log(targetId = decisionId)
-        if (!user.hasOneOfRoles(Roles.SERVICE_WORKER, Roles.ADMIN)) {
+        if (!user.hasOneOfRoles(Roles.SERVICE_WORKER, Roles.ADMIN, Roles.UNIT_SUPERVISOR)) {
```

Comment:
I might be wrong here but shouldn't unit supervisor only see decision made to her unit? This implementation gives supervisor permission to see every decision.

Reply:
Same as with applications: managers need to know the UUID of the document in order to see it, and they can only see the UUIDs of the documents related to their own units. But yes, probably better limit the access to unit's documents' only